specs/experimental/admin-and-auth.md

```diff
@@ -1,9 +1,12 @@
-# Security Council Safe
+# Superchain Administration and Authorization Model
```

Comment:
Maybe a nit, but is there a difference you are trying to convey between "Administration" and "Authorization"? Wondering if we should shorten this to 
```suggestion
# Superchain Administration Model
```